lib/board.rb

```diff
@@ -84,4 +84,8 @@ def diagonal_arrays
     end
     diagonal_arrays
   end
-end
\ No newline at end of file
+
+  def reset
```

Comment:
Thank you for trying it with a new object first. This works much better

Reply:
haha yeah, it doesn't feel as convenient(?) but it works!